src/azure-cli/azure/cli/command_modules/acr/custom.py

```diff
@@ -150,6 +153,11 @@ def acr_update_custom(cmd,
 def _configure_public_network_access(cmd, registry, enabled):
     PublicNetworkAccess = cmd.get_models('PublicNetworkAccess')
     registry.public_network_access = (PublicNetworkAccess.enabled if enabled else PublicNetworkAccess.disabled)
```

Comment:
We can remove this line now?

Reply:
no this is also to change from a boolean value to the string actually used by the RP

Reply:
sorry I don't understand how is this line helping, you are anyway overriding the value in line 159 and 161. What am I missing?